### PR TITLE
feat: Tab -> add validity indicator

### PR DESCRIPTION
**Pull request by sofa-tata**

### CODE REVIEW

File: src/components/Tab/tab.scss [hunk 1]
```diff
@@ -30,7 +30,16 @@
     }
   }
 
+  .tab-title-wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-between;
+  }
```
Comment (mvoloshin0458) [inline src/components/Tab/tab.scss:39]: When it's not a side tab, the content will be centered, and items will be too close to each other. Let's add a gap between the title and the indicators for that case
image
image

File: src/components/Tab/ValidityIndicator/validityIndicator.scss [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.validity-indicator {
+  width: 32px;
```
Comment (mvoloshin0458) [inline src/components/Tab/ValidityIndicator/validityIndicator.scss:2]: It looks perfect for the two digits but feels empty with one digit. Let's do the size dynamic
image

File: src/components/Tab/Tab.tsx [hunk 3]
```diff
@@ -33,7 +35,8 @@ function Tab(props: TabProps) {
     isSubTab = false,
     navigateTo,
     info,
-    hasIncompleteFields
+    hasValidIndicator = false,
+    unfilledFields
```
Comment (mvoloshin0458) [inline src/components/Tab/Tab.tsx:39]: The component cares only about the length of the `unfilledFields` and not the content; let's pass only the length

Comment (mvoloshin0458) [inline src/components/Tab/Tab.tsx:39]: Let's make the name a bit more generic, e.g. `invalidFieldsLength` or something like that

**Review by mvoloshin0458 — APPROVED** [review]

File: src/components/Tab/ValidityIndicator/validityIndicator.scss (hunk 2 above)
Comment (gonenidan) [reply src/components/Tab/ValidityIndicator/validityIndicator.scss:2]: I think it's better to have all the same size
it's look aligned that way

File: src/components/Tab/Tab.tsx (hunk 3 above)
Comment (sofa-tata) [reply src/components/Tab/Tab.tsx:39]: I'm planning to add some more content later on which does depend on the content, so will need it

**Review by mvoloshin0458 — APPROVED** [review]

**Pull request merged**